### DISCUSSION TITLE
Create lists of the things that have been done

### DISCUSSION THREAD

**Discussion by Lerica6** [iSoron/uhabits#1391] [Feature Requests] [answered]
Hi, I've just discovered your app and find it great. However, in the "yes or no" goals I would love to be able to add lists of the things I have achieved. 
For example : write down the title of books or movies I've seen, or simply name the more specific kind of task I've been doing that day in my category "exercice".
Hope my proposal is clear and doable ^^ 
Thank you !

**Comment by hiqua** [comment, marked as answer]
That's already implemented in the upcoming version: you'll be able to add notes.

**Comment by Lerica6** [comment]
Great ! Thank you very much !

Le dimanche 5 juin 2022, Quentin Hibon ***@***.***> a écrit :

> That's already implemented in the upcoming version: you'll be able to add
> notes.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/iSoron/uhabits/discussions/1391#discussioncomment-2886642>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZMJGJXUE6XAQUEYMYD7J43VNTFZLANCNFSM5XH3ZE7A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Claire Frament*

*+32(0)494.69.58.32*


* ~~~*

*Actualités :*
*Lili sous la pluie, avec le Kusfi, prix Kiwanis aux Rencontres de Huy 2021**
~ Tournée à suivre*
*Mini-Ver, **avec l'Anneau Théâtre, 31mars, 1er et 2 avril à la Vénerie /
du 22 au 25 mai à Ekla **et plusieurs autres dates... ~ Tournée à suivre*
*La Pastorale du Périph', avec Lieu Commun - Résidences de création *
*"A pas de Forêt", *
*avec le Centre Culturel de la Vénerie - Artiste collaboratrice*
*Festival interculturel du conte à Bruxelles avec **Source Plurielle - **
partenaire** et conteuse*

*MoMatelier, atelier théâtre pour les 8-11 ans à la Montagne Magique*

*Les Pouces Verts (titre provisoire), résidences de recherche - porteuse de
projet*

*"mieux vaut allumer une bougie que maudire les ténèbres"*
